packages/central-server/src/apiV2/feedItems/CreateFeedItems.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { BESAdminCreateHandler } from '../CreateHandler';
+import { uploadImage } from '../utilities';
+/**
+ * Handles POST endpoints:
+ * - /feedItems
+ */
+
+export class CreateFeedItems extends BESAdminCreateHandler {
+  async createRecord() {
+    const {
+      country_id,
+      permission_group_id,
+      creation_date,
+      template_variables: { title, image, body, link },
+      type,
+      geographical_area_id,
+      user_id,
+      record_id,
+    } = this.newRecordData;
+
+    await this.models.wrapInTransaction(async () => {
+      const newFeedItem = await this.models.feedItem.create({
```

Comment:
Hmm. I'm not 100% sure what the behaviour would be here, but you're not using `wrapInTransaction` in our standard way (using `transactingModels`). I'm not sure if using `this.models` inside the `wrapInTransaction` block is going to correctly run as a db transaction 🤔

```suggestion
    await this.models.wrapInTransaction(async transactingModels => {
      const newFeedItem = await transactingModels.feedItem.create({
```